diff --git a/services/admin/config/application.rb b/services/admin/config/application.rb
--- a/services/admin/config/application.rb
+++ b/services/admin/config/application.rb
@@ -10,6 +10,7 @@ module Admin
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 8.0
+    config.action_controller.allow_forgery_protection = false

     # Please, add to the `ignore` list any other `lib` subdirectories that do
     # not contain `.rb` files, or that should not be reloaded or eager loaded.
